src/test/resources/net/sf/jabref/importer/fileformat/MedlineImporterTestSingleEntry.xml

```diff
@@ -1,4 +1,3 @@
-<PubmedArticleSet>
 <PubmedArticle>
 <MedlineCitation Owner="NLM" Status="MEDLINE">
 <PMID Version="1">1</PMID>
```

Comment:
I weren't sure how to handle this. I thought that the PMID with the highest version should be added, since it is the newest version. Should the PMID with Version = 1 also be added? For example as pmid-version1?